### ISSUE TITLE
roundedRect mask produces incorrect overlay

### ISSUE THREAD

**Issue by danielsaidi** [guoyingtao/Mantis#121]
When using `roundedRect` as `cropShapeType`, the overlay incorrect...or at least not as expected.

If you add this to the SwiftUI demo:

```
Button("Rounded Corner Rect") {
    reset()
    cropShapeType = .roundedRect(radiusToShortSide: 0.1, maskOnly: true)
    showingCropper = true
}.font(.title)
```

The cropper looks like this:

![image](https://user-images.githubusercontent.com/429927/127784280-654d5edc-c64d-48bf-868c-67c98fc88de9.png)

As you can see, the rounded corners is stretched out, which is not what I expected.

**Comment by guoyingtao** [comment]
@danielsaidi Thanks for reporting this issue. Can you help to check if this commit solves the problem?
```Ruby
pod 'Mantis', :git => 'https://github.com/guoyingtao/Mantis.git', :commit => 'a5600dc4ad263164d7ad32b81db8be003a74d03d'`
```

**Comment by danielsaidi** [comment]
@guoyingtao It works like a dream! 🤩 

![image](https://user-images.githubusercontent.com/429927/128145164-0d305e5f-89e1-45e7-95c6-c3490cae78ed.png)

**Comment by guoyingtao** [comment]
Thanks for the tests!

**Comment by danielsaidi** [comment]
Is it possible to create a new patch with this fix?

**Comment by guoyingtao** [comment]
Sure, I will do it tonight.

**Comment by danielsaidi** [comment]
Great!

I also posted an issue regarding Xcode 13. Will post some updates there now, after trying to get it to work without success :(

**Comment by guoyingtao** [comment]
I released Mantis 1.7.2 which includes this fix.

**Comment by danielsaidi** [comment]
I can verify that it works great, thank you! 👍
